msteams-platform/bots/how-to/conversations/conversation-messages.md

```diff
@@ -215,6 +223,309 @@ async def on_members_added_activity(
 
 Messages sent between users and bots include internal channel data within the message. This data allows the bot to communicate properly on that channel. The Bot Builder SDK allows you to modify the message structure.
 
+## Update message
+
+When you edit or Undo delete a message sent to a bot in a chat, the bot gets a notification of the edit or undo delete an event.
+
+To get an edit or undo delete event notification for a sent message in a bot, you can override the `OnTeamsMessageEditAsync`, `OnTeamsMessageUndeleteAsync`, or `OnMessageUpdateActivityAsync` handlers.
```

Comment:
To get an edit or undo delete event notification for a sent message in a bot, you can override the `OnTeamsMessageEditAsync`, `OnTeamsMessageUndeleteAsync`, or `OnMessageUpdateActivityAsync` handlers.

WIll it better to edit to:
To get an edit or undelete message event notification in a bot, you can override the 1) `OnTeamsMessageEditAsync`, `OnTeamsMessageUndeleteAsync`, or 2) `OnMessageUpdateActivityAsync` handlers. 1) is suggested.

Reply:
Updated the doc as suggested